src/ccstruct/normalis.cpp

```diff
@@ -182,8 +181,8 @@ static void ComputeRunlengthImage(const TBOX &box,
   // Now set the image pixels the the MIN of the x and y runlengths.
   for (int iy = 0; iy < height; ++iy) {
     int x = 0;
-    for (int i = 0; i < x_coords[iy].size(); ++i) {
-      int x_edge = ClipToRange(x_coords[iy][i], 0, width);
+    for (auto x_coord : x_coords[iy]) {
```

Comment:
same as prev

Reply:
This is also an `int` value, so no need for a reference.